test/Microsoft.Framework.Configuration.FunctionalTests/ConfigurationTests.cs

```diff
@@ -233,7 +233,38 @@ public void SetBasePathCalledMultipleTimesForEachSource()
             Assert.Equal("XmlValue3", config["XmlKey2:XmlKey4"]);
             Assert.Equal("XmlValue4", config["XmlKey2:XmlKey5:XmlKey6"]);
 
-            File.Delete(_jsonConfigFilePath);
+            File.Delete(jsonConfigFilePath);
+        }
+
+        [Fact]
+        public void GetDefaultBasePathForSources()
+        {
+            // Arrange
+            var builder = new ConfigurationBuilder();
+            var jsonConfigFilePath = Path.Combine(Directory.GetCurrentDirectory(), "test.json");
```

Comment:
Using current directory is wrong. You want to use the application base here too.